crates/rooch-types/src/framework/bitcoin_validator.rs

```diff
@@ -54,11 +54,17 @@ impl<'a> BitcoinValidatorModule<'a> {
         Ok(())
     }
 
-    pub fn rotate_authentication_key_action<V: MoveStructType>(public_key: Vec<u8>) -> MoveAction {
+    pub fn rotate_authentication_key_action<V: MoveStructType>(
+        public_key: Vec<u8>,
+        decimal_prefix_or_version: u8,
+    ) -> MoveAction {
         Self::create_move_action(
             Self::ROTATE_AUTHENTICATION_KEY_ENTRY_FUNCTION_NAME,
             vec![V::type_tag()],
-            vec![MoveValue::vector_u8(public_key)],
+            vec![
```

Comment:
`rotate_authentication_key` do not need the TypeTag argument.

Reply:
Yes. It doesn't need such type tag.